docs/api/browser-window.md

```diff
@@ -1397,12 +1397,16 @@ can still bring up the menu bar by pressing the single `Alt` key.
 
 Returns `Boolean` - Whether the menu bar is visible.
 
-#### `win.setVisibleOnAllWorkspaces(visible)`
+#### `win.setVisibleOnAllWorkspaces(visible[, visibleOnFullScreen])`
```

Comment:
`visibleOnFullScreen` should be an options-object or this will become quite a [boolean trap](https://ariya.io/2011/08/hall-of-api-shame-boolean-trap). `win.setVisibleOnAllWorkspaces(true, true)` is not readable at all. While is is `win.setVisibleOnAllWorkspaces(true, {visibleOnFullScreen: true})`.

Reply:
very true! will be fixing today!

Reply:
@paulius005 just a heads up, if you just replace it with an options object and don't make it backwards compatible with current parameter formatting then it'll be a breaking change and won't be able to go out until `v4.0.0`

Reply:
This isn't a bug fix or security fix, so it's [not eligible](https://electronjs.org/docs/tutorial/electron-versioning#beta-releases-and-bug-fixes) for the `3-0-x` branch anyway.

:+1: on @sindresorhus's suggestion of making this an options object. That's the model we followed on the menu refactor and it worked well

Reply:
updated, I used `setIgnoreMouseEvents` as an example to follow